TeleSharp.TL/TLVector.cs

```diff
@@ -11,7 +11,17 @@ namespace TeleSharp.TL
     public class TLVector<T> : TLObject, IList<T>
     {
         [TLObject(481674261)]
-        private List<T> lists = new List<T>();
+        private List<T> lists;
+
+        public TLVector(IEnumerable<T> collection)
+        {
+            lists = new List<T>(collection);
+        }
+
+        public TLVector()
+        {
+            lists = new List<T>();
+        }
```

Comment:
thanks for your contribution! can you remove this from the PR? seems unrelated with the problem at hand

Reply:
Okay, i'll solve it a bit later

Reply:
wait, I think I'm mistaken, sorry, it is related

Reply:
Well, I thought that you want to leave the default constructor empty and leave the list initialization as it was before. If there are no problems, this is very good)